src/plugins/roomslist/templates/roomslist.js

```diff
@@ -67,10 +67,44 @@ function tplRoomItem (el, room) {
         </div>`;
 }
 
+function tplRoomDomainGroup (el, domain, rooms) {
```

Comment:
This function should ideally go into a separate file. Generally we put template functions into their own files so that it's easier for 3rd party integrators to change/override them (e.g. by using webpack aliases).